packages/jest-preset-mc-app/jest-preset.js

```diff
@@ -24,6 +24,10 @@ module.exports = {
     'process.env': {
       NODE_ENV: 'test',
     },
+    // This is required for the `jose` library to work in the test environment.
+    // We use it in the packages-backend/express package.
+    // Reference: https://github.com/jestjs/jest/issues/4422#issuecomment-770274099
+    Uint8Array: Uint8Array,
```

Comment:
This was the trickiest part as I was having a very weird error but finally got some help in the linked GitHub issue.